spec/integration_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe Talkable do
-  shared_examples 'integration method' do |method_name|
+  shared_examples 'register origin method' do |method_name|
```

Comment:
makes no sense anymore if we extracted referrals methods

Reply:
changed back